src/js/demo/components/Message.jsx

```diff
@@ -1,5 +1,15 @@
 import React from "react";
-import events from "./events";
+import events from "../utils/events";
+
+function formatMessage({ line, column, message }) {
+
+    // line and column are undefined when parsing cannot occur (e.g. misconfiguration)
+    if (typeof line === "number" && typeof column === "number") {
+        return `${line}:${column} - ${message}`;
```

Comment:
This is the other bug I found. When there isn't  location information, we can just show the message!